static/app/views/performance/newTraceDetails/referrers.ts

```diff
@@ -0,0 +1,3 @@
+export enum Referrer {
```

Comment:
This is only used in a single place, can we just inline it there?

Reply:
So this is mainly for consistency between modules, where we declare all referrers in one file. For example
https://github.com/getsentry/sentry/blob/master/static/app/views/performance/cache/referrers.ts

Yes rn its only in a single place, but ideally overtime we extend this to include all referrers used in the trace view. 
Does that work for you? Otherwise I can make it inline.

Reply:
Imo, code like this is trivial to refactor and my tendency is to start simple (just less moving parts and files around which hold little value). Eod not a big deal and keeping this consistent across modules is a win.